main.go

```diff
@@ -126,7 +126,7 @@ func main() {
 	}
 
 	logger.Info("Starting elastic k8s metrics adapter...")
-	if err := cmd.Run(wait.NeverStop); err != nil {
```

Comment:
This is due to: https://github.com/kubernetes-sigs/custom-metrics-apiserver/commit/a1e8cfcf804bc5a31abb92db1930b52869c1c2d9